packages/twenty-front/src/modules/object-record/record-field/components/FieldInput.tsx

```diff
@@ -127,6 +129,12 @@ export const FieldInput = ({
         <RatingFieldInput onSubmit={onSubmit} />
       ) : isFieldSelect(fieldDefinition) ? (
         <SelectFieldInput onSubmit={onSubmit} onCancel={onCancel} />
+      ) : isFieldAddress(fieldDefinition) ? (
+        <AddressFieldInput
+          onEnter={onEnter}
```

Comment:
Not sure what im doing wrong, but the enter key is not being triggered inside of the input 🤔

Reply:
@lucasbordeau any idea on that ?

Reply:
I had to refactor a bit this component after DoubleTextInput, and create a new TextInputRaw without hotkey scopes.